torchgeo/transforms/temporal.py

```diff
@@ -68,3 +71,29 @@ def compute_transformation(
         """
         out = self.identity_matrix(input)
         return out
+
+
+class TemporalEmbedding(nn.Module):
```

Comment:
Check what other libraries call this, maybe:

Prefix:

* Periodic
* Cyclic
* Sinusoidal

Suffix:

* Embedding
* Encoder